tests/tester.js

```diff
@@ -178,7 +182,7 @@ function runTests(name, runnerArgs, cb) {
       const runner = require(`./${name}Runner.js`)
       // Tests for HFs before Istanbul have been moved under `LegacyTests/Constantinople`:
       // https://github.com/ethereum/tests/releases/tag/v7.0.0-beta.1
-      if (runnerArgs.forkConfig !== 'Istanbul') {
+      if (testGetterArgs.forkConfig !== 'Istanbul') {
```

Comment:
We should safeguard this conditional with `lteHardfork()` instead, so Berlin HF tests aren't searched for in `LegacyTests`?

Reply:
Yes, I thought about that as well, just left it out for timing reasons.